### PR TITLE
CONTRIBUTING converted to markdown file

### PR DESCRIPTION
**Pull request by pinho**
Added: CONTRIBUTING.md
Removed: CONTRIBUTING.rst

It refers to issue #1179

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# CONTRIBUTING TO COOKIECUTTER
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given. 
+
+
+    .. toctree::
```
Comment (insspb) [inline CONTRIBUTING.md]: toctree not working in markdown. Should be removed or made manually.

**Comment by insspb** [comment]
@ronaldpnh thank you for this work start. But we need to bring all text from old .rst to new .md
I will wait for the update for two-tree weeks if you will add changes that cover all old file we will merge it. 
Currently it is [Work in progress]

**Comment by insspb** [comment]
@ronaldpnh if you will not work on this, please msg here. I will close this PR in this case. Thank you.

**Comment by ronaldpnh** [comment]
Hi, don't worry, i will review and fix it.
When i finish and commit the changes, i mention you here. ;)

**Comment by ronaldpnh** [comment]
> Hi, don't worry, i will review and fix it.
> When i finish and commit the changes, i mention you here. ;)

I was verify and believe sphinx not provides a direct support to Markdown.
There is a way for this, using [recommonmark](https://recommonmark.readthedocs.io/en/latest/auto_structify.html), see [here](https://www.sphinx-doc.org/en/1.6/markdown.html)...

I am looking to do it...

File: CONTRIBUTING.md (hunk 1 above)
Comment (ronaldpnh) [reply CONTRIBUTING.md]: I was verify and believe sphinx not provides a direct support to Markdown.
There is a way for this, using [recommonmark](https://recommonmark.readthedocs.io/en/latest/auto_structify.html), see [here](https://www.sphinx-doc.org/en/1.6/markdown.html)...

I am verifying it...

**Comment by ssbarnea** [comment]
Please update or we will close the PR. Now that we fixed CI we can start merging contributions, especially simple ones like this.

**Comment by pydanny** [comment]
Let's put this in a nicer way: @pinho, please update so we can merge it in ASAP. 😀

@ssbarnea, we do our best to be nice to contributors. It's good karma. 😁

**Comment by insspb** [comment]
@pinho Thank you for your great work and efforts.
Unfortunately, same work was almost completed in #1185, and completely finished at #1216 
I am closing this commit to keep all work in one place and make it in a timely manner. 
I am inviting you to review #1216 to speed up overall progress.

**Pull request closed without merging**